src/include/homestore/btree/detail/simple_node.hpp

```diff
@@ -226,66 +232,155 @@ class SimpleNode : public VariantNode< K, V > {
 
     std::string to_string_keys(bool print_friendly = false) const override {
         // FIXME: Implement this, key may not be a unit32_t
-        return "";
-#if 0
-            std::string delimiter = print_friendly ? "\n" : "\t";
-            std::string snext = this->next_bnode() == empty_bnodeid ? "" : fmt::format("next_node={}", this->next_bnode());
-            auto str = fmt::format("{}{}.{} level:{} nEntries={} {} {} node_gen={} ",
-                                print_friendly ? "------------------------------------------------------------\n" : "",
-                                this->node_id(), this->link_version(), this->level(), this->total_entries(),
-                                (this->is_leaf() ? "LEAF" : "INTERIOR"), snext, this->node_gen());
-            if (!this->is_leaf() && (this->has_valid_edge())) {
-                fmt::format_to(std::back_inserter(str), "edge_id={}.{}", this->edge_info().m_bnodeid,
-                            this->edge_info().m_link_version);
-            }
-            if (this->total_entries() == 0) {
-                fmt::format_to(std::back_inserter(str), " [EMPTY] ");
-                return str;
+        //        return "";
+        // #if 0
```

Comment:
Please return it back` if 0 ` to not break homeobject . I will fix it soon to work for it too, soon